### ISSUE TITLE
Does not open the call when tapping on the persistent notification of a minimized call on Android

### ISSUE THREAD

**Issue by saibotma** [jitsi/jitsi-meet-flutter-sdk#74]
## Steps to reproduce:
- Run the example code below on an Android phone.
- Tap on "Join"
- Minimize the call
- Close the PiP window
- Tap on the persistent notification
- Notice that nothing happens

Works using https://pub.dev/packages/jitsi_meet_wrapper.

## Video:

https://github.com/user-attachments/assets/2f6bced0-00ae-438e-9b40-c7946eaa7c8c

## Example:
```dart
import 'package:flutter/material.dart';

import 'package:jitsi_meet_flutter_sdk/jitsi_meet_flutter_sdk.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: Center(
          child: TextButton(
            onPressed: () async {
              var options = JitsiMeetConferenceOptions(
                room: "testgabigabi2",
                configOverrides: {
                  "startWithAudioMuted": true,
                  "startWithVideoMuted": true,
                },
              );
              await JitsiMeet().join(options);
            },
            child: const Text("Join"),
          ),
        ),
      ),
    );
  }
}
```

**Comment by saibotma** [comment]
It actually seems that this is not an issue of the Flutter SDK, but of the Jitsi app in general, because it also does not work on my device when using the official Jitsi app. Will create a new issue for that.

**Comment by saghul** [comment]
What version did you test? We did land a fix for it...

**Comment by saibotma** [comment]
Tested with 24.2.2 build 16757163 of the official app.

**Comment by saibotma** [comment]
Can you share the commit that fixes the issue?

**Comment by saghul** [comment]
> Tested with 24.2.2 build 16757163 of the official app.

This should be fixed in 24.3.0 which is currently rolling out. You can opt-in into betas here: https://jitsi.github.io/handbook/docs/releases#beta-versions



> Can you share the commit that fixes the issue?

I can't recall right now. @Calinteodor can you?

**Comment by Calinteodor** [comment]
I

> > Tested with 24.2.2 build 16757163 of the official app.
> 
> This should be fixed in 24.3.0 which is currently rolling out. You can opt-in into betas here: https://jitsi.github.io/handbook/docs/releases#beta-versions
> 
> > Can you share the commit that fixes the issue?
> 
> I can't recall right now. @Calinteodor can you?

Me neither, but I remember we discussed this issue. We will take a look in the upcoming days.

**Comment by saibotma** [comment]
I tested the beta version, and noticed some differences in behavior. Now, when closing or removing the PiP window, the conference opens unexpectedly. This only occurs the first time. On subsequent attempts, the call does not open, and tapping on the persistent notification also fails to open the conference.

I’ve opened a separate issue for this, as it is not related to the Flutter SDK: https://github.com/jitsi/jitsi-meet/issues/14999
